src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        BallColors[] colors = BallColors.values();
+        Random random = new Random();
+        return colors[random.nextInt(colors.length - 1)].toString();
```

Comment:
```suggestion
        return colors[random.nextInt(colors.length)].toString();
```

why `length-1`?

Reply:
NextInt does random int from 0 to N inclusive (I thought, but I was wrong - read the dock) :)